src/main/scala/chisel3/tester/package.scala

```diff
@@ -39,11 +40,19 @@ package object tester {
         require(x.binaryPoint == value.binaryPoint, "binary point mismatch")
         pokeBits(x, value.litValue)
       }
-      case (x: Bundle, value: Bundle) => {
+      case (xx: Bundle, value: Bundle) => {
```

Comment:
Why is this `xx`?

Reply:
Probably doesn't need to be but as I was trying to figure out which vars had the information I needed it was hard because you used x in three nested contexts. All the style checkers I have seen complain about variable shadowing in these cases. Firrtl adopts the convention of just duplicating the short var name so x -> xx -> xxx through the context drops.